appveyor.yml

```diff
@@ -118,6 +121,57 @@ before_build:
 build_script:
   - call ci\appveyor-build.bat
 
+for:
+-
+  matrix:
+    only:
+      - JOB: "R without libarrow"
+  init:
+    ps: |
+          $ErrorActionPreference = "Stop"
+          Invoke-WebRequest http://raw.githubusercontent.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
```

Comment:
maybe pin the version instead of master?

Reply:
Then it wouldn't be `continuous` integration ;)

This is the standard R appveyor config; I've used it for years and have never had a problem, so I don't think there's value in pinning.